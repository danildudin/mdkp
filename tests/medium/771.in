30 8
498 252 712 293 36 726 41 47 88 653 749 782 527 579 11 903 263 691 835 707 709 15 811 422 442 294 534 91 519 242 
414 641 735 112 332 1 442 282 531 407 881 661 568 758 487 707 945 891 80 968 422 544 628 477 51 76 282 769 476 799 6967
65 785 965 285 717 16 463 434 653 108 366 356 276 692 465 439 815 947 222 557 152 944 633 947 107 854 132 167 349 867 5329
625 138 692 45 204 844 648 12 211 315 364 310 194 566 522 169 312 594 367 172 970 657 200 803 984 961 404 638 430 445 7431
140 638 550 237 401 152 172 879 36 451 951 958 221 713 153 23 468 18 758 594 888 283 791 386 84 369 524 302 693 818 8312
102 129 228 42 946 115 843 988 820 785 113 38 296 511 943 753 406 159 198 79 44 547 676 174 170 3 494 191 687 803 6462
860 640 138 26 74 739 891 496 134 222 970 944 788 229 926 576 753 948 196 719 874 445 833 801 537 135 502 414 849 231 9493
2 260 392 957 889 72 86 562 373 847 875 880 41 872 393 809 29 71 971 382 16 192 826 91 767 584 601 659 168 191 6834
781 667 747 778 514 177 104 342 735 756 238 619 112 677 115 599 966 889 802 353 682 508 904 241 872 644 705 4 785 479 7510
