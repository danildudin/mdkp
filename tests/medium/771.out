9146
13
0 2 5 9 10 11 12 15 18 19 20 22 26 
